### DISCUSSION TITLE
EMS ESP node red controlling temperature heating

### DISCUSSION THREAD

**Discussion by Kudlaciks** [emsesp/EMS-ESP32#1545] [Q&A]
Hi everyone  ! 

I try to find solution but still i have no idea how to solve it. I need controll temperatur heeting from ESP EMS. I want to choose temperature of heeting in depend on temperature outside. If i try to change entities number.boiler_selflowtemp its work but only for about 5 minutes and temperature will come back to 5 degress. Currently i use to turn off my Buderus by relay but the lowest temperature is 30. I need to get lower temperatur. Pleas tell me what is the easiest way to controll Boiler by ESP EMS. I have connection with MQTT  but it is first contact with MQTT for me

**Comment by proddy** [comment]
have you looked into the documentation? https://emsesp.github.io/docs/Troubleshooting/#changing-a-value-works-at-first-but-is-then-reset-to-its-original-value

**Comment by Kudlaciks** [comment]
Thank you for your link, but I discovered if I overwrite value every one minute then it stay on 30 degrees. My question is, how do it by node red. Have you maybe some example to controll setflow temp by mqtt ?

**Comment by proddy** [comment]
Sending values by MQTT is documented in the Wiki. 

You could use the Scheduler in EMS-ESP to set the value periodically. There is a use case for later to use variables which gives you more control. https://github.com/emsesp/EMS-ESP32/issues/1423

What you could also do is ask the question on Discord or GitHub Discussions about how other people set the flow temp based on external sensors, as it's quite common, but depends on the boiler type.
